vispy/scene/cameras/base_camera.py

```diff
@@ -108,6 +109,19 @@ def depth_value(self, value):
             raise ValueError('depth value must be positive')
         self._depth_value = value
 
+    @property
```

Comment:
Maybe `mouse_translation_factor`? It seems like they should be different properties at least.